pages/acceptInvite.page.tsx

```diff
@@ -0,0 +1,132 @@
+import Head from 'next/head';
+import { useRouter } from 'next/router';
+import React, { ReactElement, useEffect, useRef } from 'react';
+import { useSnackbar } from 'notistack';
+import { useTranslation } from 'react-i18next';
+import { SetupPage } from 'src/components/Setup/SetupPage';
+import { useAccountListId } from 'src/hooks/useAccountListId';
+import useGetAppSettings from 'src/hooks/useGetAppSettings';
+import { useRequiredSession } from 'src/hooks/useRequiredSession';
+import { loadSession } from './api/utils/pagePropsHelpers';
+
+interface FetchAcceptInviteProps {
+  apiToken: string;
+  url: string;
+  inviteType: string;
+  id: string;
+  code: string;
+}
+
+export const fetchAcceptInvite = ({
+  apiToken,
+  url,
+  inviteType,
+  id,
+  code,
+}: FetchAcceptInviteProps): Promise<Response> => {
+  return fetch(process.env.REST_API_URL + url, {
+    method: 'PUT',
+    headers: {
+      authorization: `Bearer ${apiToken}`,
+      'content-type': 'application/vnd.api+json',
+      accept: 'application/vnd.api+json',
```

Comment:
I don't think that the `Accept` header is necessary. None of our other fetch calls to the API server add it.